start_bundle_coupled.R

```diff
@@ -203,14 +203,14 @@ if ("max_iterations" %in% colnames(scenarios_coupled)) {
 }
 message("max_iterations:        ", max_iterations)
 
-common <- intersect(rownames(settings_remind),rownames(scenarios_coupled))
+common <- intersect(rownames(settings_remind), rownames(scenarios_coupled))
 knownRefRuns <- apply(expand.grid(prefix_runname , common, "-rem-", seq(max_iterations)), 1, paste, collapse="")
 if (! identical(common, character(0))) {
   message("\n################################\n")
   message("The following ", length(common), " scenarios will be started:")
   message("  ", paste(common, collapse = ", "))
 } else {
-  stop("No scenario found with start=", startgroup, " in ", basename(path_settings_coupled), ".")
+  stop("No scenario found with start=", startgroup, " in ", basename(path_settings_coupled), " that is also defined in ", basename(path_settings_remind), ".")
```

Comment:
added that because the error message was not explaining the actual problem in one case